### ISSUE TITLE
Prelude overrides important global default binding for C--

### ISSUE THREAD

**Issue by AtomicNess123** [bbatsov/prelude#1354]
## Expected behavior
`C--` is a global binding for providing a specific kind of prefix argument in Orgmode.

## Actual behavior
Prelude just wipes that out: It was defined in `prelude-global-keybindings.el` to `text-scale-decrease`, so I commented this line. 

## Steps to reproduce the problem
Just press `C--` 

## Environment & Version information
Prelude version  1.1.0 (2021-02-14)

### Emacs version
Emacs 27.2

### Operating system
MacOS  10.15.7 Catalina

**Comment by bbatsov** [comment]
Yeah, probably this keybinding was a bad idea. I think I added it as a helper for people coming from other editors, but I'd be fine to wipe the text-scale keybindings completely from Prelude.

**Comment by AtomicNess123** [comment]
> Yeah, probably this keybinding was a bad idea. I think I added it as a helper for people coming from other editors, but I'd be fine to wipe the text-scale keybindings completely from Prelude.

Any implementation?
